Mathlib/Algebra/Algebra/Equiv.lean

```diff
@@ -196,18 +196,22 @@ theorem coe_ringEquiv_injective : Function.Injective ((↑) : (A₁ ≃ₐ[R] A
   fun _ _ h => ext <| RingEquiv.congr_fun h
 #align alg_equiv.coe_ring_equiv_injective AlgEquiv.coe_ringEquiv_injective
 
+@[deprecated map_add (since := "2024-06-20")]
```

Comment:
Should this be `_root_.map_add`? Reading just the code, I cannot tell why this shouldn't have a `_root_`, but e.g. `map_smul` should. Same for the next three items.

Reply:
You need `_root_.` here or `protected` on the deprecated lemma - either is enough to make lean pick up the correct declaration.